python/lsst/jointcal/jointcal.py

```diff
@@ -277,32 +363,46 @@ def run(self, dataRefs, profile_jointcal=False):
                 print("still some ouliers but chi2 increases - retry")
             else:
                 break
-                print("unxepected return code from Minimize")
+                print("unxepected return code from minimize")
 
-        # Fill reference and measurement n-tuples for each tract
-        tupleName = "res_" + str(dataRefs[0].dataId["tract"]) + ".list"
-        fit.MakeResTuple(tupleName)
+        return fit, model, sky2TP
```

Comment:
I'd vote for the namedtuple option if it's entirely internal and you don't need specific features of pipe_base.